compose/opensearch.yml

```diff
@@ -0,0 +1,37 @@
+version: "3"
+services:
+  opensearch-node:
+    image: docker.io/opensearchproject/opensearch:latest
+    hostname: opensearch-node
+    environment:
+      discovery.type: single-node
```

Comment:
That's an odd looking environment variable. This is something OpenSearch wants specifically in this format?

Reply:
I think so, https://opensearch.org/docs/latest/install-and-configure/configuring-opensearch/index/

They represent setting fields. I can try with `OPENSEARCH_DISCOVERY_TYPE`...

Reply:
Nope, that does not work! Those setting fields translate directly/exactly into the environment variables.

Reply:
Looks like it is not yet possible: https://github.com/opensearch-project/opensearch-build/issues/4451